examples/contrib/save_streamed_data.py

```diff
@@ -58,7 +58,8 @@ def __call__(self, data):
             return data
 
         if not self.fh:
-            self.path = datetime.fromtimestamp(self.flow.request.timestamp_start).strftime(ctx.options.save_streamed_data)
+            self.path = datetime.fromtimestamp(self.flow.request.timestamp_start).strftime(
+                ctx.options.save_streamed_data)
```

Comment:
is this auto-formatting or did you feel today is a day for shorter lines?

Reply:
This is auto-formatting...